### ISSUE TITLE
Question about scheduler state and resuming training.

### ISSUE THREAD

**Issue by Farabi-shafkat** [jfzhang95/pytorch-video-recognition#51]
I was wondering why only the optimezer state is being saved but the scheduler state is not being saved. Is this a bug? If I want to resume training shouldn't I save the model + optimizer + scheduler state?
